lib/vra/client.rb

```diff
@@ -93,19 +93,17 @@ def authorized?
       return false if @bearer_token.value.nil?
 
       response = http_head("/identity/api/tokens/#{@bearer_token.value}", :skip_auth)
-      if response.code == 204
-        true
-      else
-        false
-      end
+      response.success?
```

Comment:
This does depend on how well-behaved vrealize is :-)  In theory, a client ought to be able to treat any 2XX response as a success, but I'm happy enough restricting it to the spec if we don't want to make that assumption.